ArcGISRuntimeSDKQt_CppSamples/Features/ControlTimeExtentTimeSlider/ControlTimeExtentTimeSlider.qml

```diff
@@ -0,0 +1,47 @@
+// [WriteFile Name=ControlTimeExtentTimeSlider, Category=Features]
+// [Legal]
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import Esri.ArcGISRuntime.Toolkit 100.13
+import Esri.Samples 1.0
+
+Item {
+
+    // add a mapView component
+    MapView {
+        id: view
+        anchors.fill: parent
+        focus: true
+
+        // Add a TimeSlider from the toolkit to the MapView
+        TimeSlider {
+                    anchors {
+                        left: parent.left
+                        right: parent.right
+                        bottom: parent.bottom
+                    }
+
+                    geoView: view
+                }
```

Comment:
indentation of this is off. ctrl + a followed by ctrl + i should auto indent everything for you in Qt Creator